features/dd-sdk-android-session-replay/src/test/kotlin/com/datadog/android/sessionreplay/forge/SessionReplayConfigurationForgeryFactory.kt

```diff
@@ -27,7 +28,11 @@ class SessionReplayConfigurationForgeryFactory : ForgeryFactory<SessionReplayCon
             customOptionSelectorDetectors = forge.aList { mock() },
             startRecordingImmediately = forge.aBool(),
             sampleRate = forge.aFloat(min = 0f, max = 100f),
-            dynamicOptimizationEnabled = forge.aBool()
+            dynamicOptimizationEnabled = forge.aBool(),
+            systemRequirementConfiguration = SystemRequirementConfiguration.Builder()
```

Comment:
maybe create a dedicated factory for the `SystemRequirementConfiguration`? Not sure if it will be useful elsewhere though.

Reply:
Since it has only two attributes, a dedicated factory may be a bit overkill?

Reply:
It is rather a question if we need to create it elsewhere in the tests (now or in the future)

Reply:
I don't use it elsewhere for the moment, and probably not in the future since the whole feature is developed and covered by tests